src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -305,9 +306,8 @@ private void displaySlimefunItem(ChestMenu menu, ItemGroup itemGroup, Player p,
                             Slimefun.getLocalization().sendMessage(pl, "guide.cheat.no-multiblocks");
                         } else {
                             ItemStack clonedItem = sfitem.getItem().clone();
-
                             if (action.isShiftClicked()) {
-                                clonedItem.setAmount(clonedItem.getMaxStackSize());
+                                clonedItem.setAmoutMaxStackSize();
```

Comment:
Looks like a typo? No idea how you got this to build locally?
```suggestion
                                clonedItem.setAmountMaxStackSize();
```

Reply:
I am assuming it’s a typo and the method isn’t actually called that?

Reply:
nvm it for some reason didnt pull all changes only a few of this PR
My git has been acting up lately on my new laptop

Reply:
This can just be replaced with `clonedItem.setAmount(clonedItem.getMaxStackSize());` and this should then build? I’m interested to know where this method came from, I doubt it was just… made up?

Reply:
All these changes stam of from 2021 so it can very well just be old code that somehow found its way in the merge conflics

Reply:
Think this is meant to be
```suggestion
                                clonedItem.setAmount(clonedItem.getMaxStackSize());
```